mmv1/third_party/terraform/envvar/envvar_utils.go

```diff
@@ -146,6 +150,10 @@ func GetTestUniverseDomainFromEnv(t *testing.T) string {
 	return transport_tpg.MultiEnvSearch(UniverseDomainEnvVars)
 }
 
+func GetProjectPrefixFromEnv() string {
```

Comment:
Can you document context / why this needs to be used?

Reply:
Will do